docs/source/execute.rst

```diff
@@ -50,7 +50,7 @@ where the working tree looks like this::
       |-- f0.csv -> ../.git/annex/objects/[...]
       `-- f1.csv -> ../.git/annex/objects/[...]
 
-The clean.py script takes two positional arguments (e.g., ``./clean.py
+The ``clean.py`` script takes two positional arguments (e.g., ``./clean.py
```

Comment:
I don't have a strong preference on whether we should use code tags for file names, but we should use a consistent style across the document.  There is another clean.py instance in this same file.  I'll update it.